.circleci/config.yml

```
@@ -1,27 +1,71 @@
 ---
 version: 2.1
+
 orbs:
   slack: circleci/slack@4.1.3
+
+executors:
+  pre-commit-executor:
+    docker:
+      - image: quay.io/astronomer/ci-pre-commit:2022-06
```

Might as well keep this fresh.

```suggestion
      - image: quay.io/astronomer/ci-pre-commit:2022-08
```